src/Integration.UnitTests/Binding/DotNetBindingConfigProviderTests.cs

```diff
@@ -24,15 +24,11 @@
 using System.Threading;
 using System.Threading.Tasks;
 using FluentAssertions;
-using Microsoft.VisualStudio.CodeAnalysis.RuleSets;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
-using NuGet;
 using SonarLint.VisualStudio.Integration.Binding;
-using SonarLint.VisualStudio.Integration.NewConnectedMode;
 using SonarLint.VisualStudio.Integration.Resources;
 using SonarQube.Client;
-using SonarQube.Client.Messages;
 using SonarQube.Client.Models;
 using Language = SonarLint.VisualStudio.Core.Language;
 using NuGetPackageInfo = SonarLint.VisualStudio.Core.CSharpVB.NuGetPackageInfo;
```

Comment:
There's a naming clash between the `RuleSet` in Core and the VS `RuleSet`. I'm hoping it will occur less often after some of the later changes.
If it's still a problem we can consider renaming our version.